datahub-web-react/src/app/entity/dataset/profile/schema/components/SchemaHeader.tsx

```diff
@@ -1,26 +1,28 @@
 import React from 'react';
 import { useHistory, useLocation } from 'react-router-dom';
-import { Button, Popover, Select, Tooltip, Typography } from 'antd';
+import { Button, Input, Popover, Select, Tooltip, Typography } from 'antd';
+import { debounce } from 'lodash';
 import {
     AuditOutlined,
     CaretDownOutlined,
     FileTextOutlined,
     QuestionCircleOutlined,
+    SearchOutlined,
     TableOutlined,
 } from '@ant-design/icons';
-import styled from 'styled-components';
+import styled from 'styled-components/macro';
```

Comment:
whats up with this?

Reply:
this is what I mentioned to you some time back where if you import from `/macro` and look in the browser dev tools and check out the html, styled component names are shown, making it much easier to find what styled component is what